lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -210,6 +212,7 @@ export class FormFieldModel extends FormWidgetModel {
             this.schemaDefinition = json.schemaDefinition;
             this.precision = json.precision;
             this.externalProperty = json.externalProperty;
+            this._value = this.parseValue(json);
```

Comment:
any particular reason the parse was moved ?

Reply:
It bases on previously calculated values so it's better to have it at the end, some new tests failed because of that